api/tsyringe.config.ts

```diff
@@ -1,8 +1,13 @@
 import 'reflect-metadata'
 import { PrismaClient } from '@prisma/client'
+import { RedisClient, createClient } from 'redis'
 
 import { container, instanceCachingFactory } from 'tsyringe'
 
 container.register<PrismaClient>(PrismaClient, {
   useFactory: instanceCachingFactory<PrismaClient>(() => new PrismaClient()),
 })
+
+container.register<RedisClient>(RedisClient, {
+  useFactory: instanceCachingFactory<RedisClient>(() => createClient()),
+})
```

Comment:
```suggestion
container.register<RedisClient>(RedisClient, {
  useFactory: instanceCachingFactory<RedisClient>(() => createClient({
    host: process.env.REDIS_HOST,
    port: process.env.REDIS_PORT  
  })),
})
```
or something like this maybe?

Reply:
Thanks and sorry for spamming

Reply:
I'm not getting this error on my local machine 
<img width="822" alt="image" src="https://user-images.githubusercontent.com/62339705/120187375-554a0600-c232-11eb-8538-46638c387917.png">

Reply:
I had this too, it comes from the recent update to apollo server. Make sure to run `yarn install`. I also had to restart VS for some reason. If that also doesn't work you can try to run `yarn upgrade`.